docs/development/extensions-contrib/iceberg.md

```diff
@@ -0,0 +1,121 @@
+---
+id: iceberg 
+title: "Iceberg"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+## Iceberg Ingest Extension
+
+This extension provides [IcebergInputSource](../../ingestion/input-sources.md#iceberg-input-source) which enables ingestion of data stored in the Iceberg table format into Druid.
```

Comment:
```suggestion
The Iceberg extension lets you ingest data stored in the Iceberg table format into Apache Druid.
```
Consider removing [IcebergInputSource](../../ingestion/input-sources.md#iceberg-input-source) from this introductory sentence. Instead, link to the Iceberg table format.